bundles/org.openhab.binding.openwebnet/src/main/resources/OH-INF/thing/channels.xml

```diff
@@ -113,6 +113,16 @@
 		<state pattern="%.1f %unit%" step="0.5"/>
 	</channel-type>
 
+	<channel-type id="targetTemperature" advanced="true">
```

Comment:
What is the difference between setpointTemperature and targetTemperature ?

Reply:
- `setpointTemperature` -> The zone or Central Unit manual setpoint temperature
- `targetTemperature` ->  The current zone target temperature according to `mode`, `setpointTemperature` and `localOffset`.

For example one thermostat can have the `setpointTemperature` set to 22° ad the `locaIOffset` (physical knob) set to "-1" so `targetTemperature` would be 21°.
Another example is `setpointTemperature`=22° but thermostat is locally set to "anti-freeze", so `targetTemperature` would be 7°.

Reply:
Understood.